pkg/authorization/registry/localsubjectaccessreview/rest_test.go

```diff
@@ -259,7 +259,7 @@ func (r *subjectAccessTest) runTest(t *testing.T) {
 
 	expectedResponse := &authorizationapi.SubjectAccessReviewResponse{
 		Namespace:       r.reviewRequest.Action.Namespace,
-		Allowed:         r.authorizer.allowed,
+		Allowed:         r.authorizer.allowed == kauthorizer.DecisionAllow,
```

Comment:
`Denied: r.authorizer.allowed == kauthorizer.DecisionDeny,`

Reply:
> Denied: r.authorizer.allowed == kauthorizer.DecisionDeny,

Not for our object.